rmw_implementation/package.xml

```diff
@@ -11,11 +11,20 @@
 
   <build_depend>rcutils</build_depend>
   <build_depend>rmw</build_depend>
+  <build_depend>rmw_connext_cpp</build_depend>
+  <build_depend>rmw_fastrtps_cpp</build_depend>
+  <build_depend>rmw_opensplice_cpp</build_depend>
```

Comment:
Why should these be added upstream? The group dependency below already covers them.

Reply:
They're redundant with upstream's use of the group tag so they don't change upstream behavior. The other dependency work for this repository will happen after package.xmls are handled so with these changes upstream no package.xml overlay is needed in the release repo.

But since they are redundant I can instead add them to an overlay or patch in the release repositor if that is preferred..

Reply:
It makes sense to add them here is that avoid patching the manifest in the release repo since it doesn't affect the behavior.

Please add a comment above these three line describing why they are in there.